### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by liu-shaojun**
### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the Unit test [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-unit-tests.yml) by inputting the full commit ID (SHA value) of your last commit (e.g., `a1b2c3d4e5f6g7h8i9j0`). And paste your action link here.
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Comment by liu-shaojun** [comment]
PR Validation: https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/9753926803

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ### 4. How to test?
 - [ ] N/A
-- [ ] Unit test: Please manually trigger the Unit test [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-unit-tests.yml) by inputting the full commit ID (SHA value) of your last commit (e.g., `a1b2c3d4e5f6g7h8i9j0`). And paste your action link here.
+- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-unit-tests.yml) by inputting the full commit ID (SHA value) of your last commit (e.g., `a1b2c3d4e5f6g7h8i9j0`). And paste your action link here.
```
Comment (glorysdj) [inline .github/pull_request_template.md:22]: And paste your action link here once it's successfully finished.

**Comment by liu-shaojun** [comment]
ALL TEST PASSED: https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/9771086304

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**